src/app/components/menu/menu.component.ts

```diff
@@ -0,0 +1,38 @@
+import {Component, EventEmitter, OnInit, Output} from '@angular/core';
+import {GenresHolderService} from "../../services/genres/genres-holder.service";
+import {SortService} from "../../services/sortFunctions/sort.service";
+import {ISelected} from "../../services/sortFunctions/iselected";
+import {Movie} from "../../model/movie/movie";
+
+@Component({
+  selector: 'app-menu',
+  templateUrl: './menu.component.html',
+  styleUrls: ['./menu.component.scss']
+})
+export class MenuComponent implements OnInit {
```

Comment:
it's better to have MenuComponent as only presentation component, it displays data and send events when something happens. Sorting logic should be in movie service. This component should only output the item which was selected, so the parent component, that handles some logistics, will direct the event to a service